docs/make.jl

```diff
@@ -2,16 +2,11 @@ using Documenter
 using Literate, JSON
 using COBREXA
 
-# Note: required to deploy the doc from Gitlab CI instead of Travis
-ENV["TRAVIS_REPO_SLUG"] = "LCSB-BioCore/COBREXA.jl"
-ENV["TRAVIS_BRANCH"] = "master"
-
-# set the merge/pull request ID
-if "CI_EXTERNAL_PULL_REQUEST_IID" in keys(ENV)
-    ENV["TRAVIS_PULL_REQUEST"] = ENV["CI_EXTERNAL_PULL_REQUEST_IID"]
-else
-    ENV["TRAVIS_PULL_REQUEST"] = "false"
-end
+# some settings
+dev_docs_folder = "dev"
+pages_branch = "gh-pages"
+github_repo_slug = "LCSB-BioCore/COBREXA.jl"
```

Comment:
this should be retrievable from the env

Reply:
Doesn't seem so, I'd need to parse it out of EXTERNAL_REPO.

Reply:
this script executes on gitlab CI, so we can use `CI_PROJECT_NAMESPACE` and `CI_PROJECT_NAME`

Reply:
but that doesn't match the github ones (that I need) right?

Reply:
it is the same as the gitlab one, as it is a mirror

Reply:
ok, fixed in #500